examples/gin/README.md

```diff
@@ -2,10 +2,28 @@
 
 This is an example of basic Gin app with Airbrake middleware that reports route stats.
 
-## How to run API
+## How to run Example API
+
+Copy the example in your testing(eg. ginbrake) folder.
+
+Insert your project ID and project key in main.go file
+
+run:
 
 ```bash
-go run *.go -env=production -airbrake_project_id=123456 -airbrake_project_key=FIXME
+mkdir ginbrake
```

Comment:
Why do we need to create a new `ginbrake` directory? Can the user just:

1. Update `main.go` to include project ID and key
2. Initialize go mod
3. Run main.go

```bash
# Add project ID and key to main.go
go mod init
go mod tidy
go run main.go
```

Reply:
This is to give developers the choice not to clone the whole repository and just the example application. Otherwise they would have to clone the repository or find where the gobrake package is installed and traverse to gin sample application.

Reply:
How can you clone just the example application?